pulpcore/cli/core/generic.py

```diff
@@ -71,6 +94,52 @@ def _task_group_filter_callback(
         help=_("List only tasks in this task group. Provide pulp_href or UUID."),
         callback=_task_group_filter_callback,
     ),
+    click.option(
+        "--parent-task",
+        help=_("Parent task by uuid or href."),
+        callback=_task_filter_callback,
+    ),
+    resource_option(
+        "--worker",
+        default_plugin="core",
+        default_type="none",
+        context_table={"core:none": PulpWorkerContext},
+        href_pattern=PulpWorkerContext.HREF_PATTERN,
+        help=_("Worker used to execute the task by name or href."),
+    ),
+    click.option(
+        "--created-resource",
+        "created_resources",
+        help=_("Href of a resource created in the task."),
+    ),
+    click.option(
+        "--reserved-resources-record",
```

Comment:
reserved_resources expects an array of resource-ids - multiple=True

Reply:
It looks complicated. Do we absolutely need it today? I'd be happy waiting for a real user request.

BTW: We have a `pulp file repository task list` command that can (almost?) do the same.